src/sentry/api/helpers/group_index.py

```diff
@@ -420,6 +420,22 @@ def self_subscribe_and_assign_issue(acting_user, group):
             return Actor(type=User, id=acting_user.id)
 
 
+def track_update_groups(function):
+    def wrapper(request, *args, **kwargs):
+        response = function(request, *args, **kwargs)
+
+        serializer = GroupValidator(data=request.data, partial=True)
+        results = dict(serializer.validated_data) if serializer.is_valid() else {}
+        tags = {key: True for key in results.keys()}
```

Comment:
I'm not sure if this is the best way to do this, but using the validated data to check what kind of action this is, ie. bookmarking or changing user etc.

Reply:
So, being not super familiar with the codebase, feel free to tell me to pound sand.

That said, it looks like you are valuing decoration and organization over the additional costs of deserialization. Do you know what type of impact this would be? For example, how fast is this endpoint today (please post a discover link ;-) ) and what would be the timing of deserialization in the worst case (bulk action with 20ish actions)?

Ultimately, I don't think it will be all that impactful, but that's a gut feel and providing some data to generally align or disprove shouldn't be too onerous.

Reply:
😢 so the discover link for this has a bug, the transaction duration for methods other than a `GET` are coming back as `n/a` query [here](https://sentry.io/organizations/sentry/discover/results/?field=transaction&field=transaction.duration&field=project&field=http.method&field=event.type&name=All+Events&query=transaction%3A%2Fapi%2F0%2Forganizations%2F%7Borganization_slug%7D%2Fissues%2F+http.method%3APUT&sort=transaction.duration&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1).

But to assuage some worry about doing an extra serializer call, we already do 1 serialize per project [here](https://github.com/getsentry/sentry/pull/17542/files#diff-9577ad7c55d768a4da4a28e48d27c522R455-R458), so the the difference in runtime should be small.